detekt.yml

```diff
@@ -2,24 +2,35 @@ config:
   warningsAsErrors: true
 
 complexity:
+  ComplexMethod:
+    threshold: 17
+  LongMethod:
+    threshold: 70
   LongParameterList:
     # It is suggested to increase this for Compose: https://detekt.dev/docs/introduction/compose/#longparameterlist
     functionThreshold: 8
     ignoreDefaultParameters: true
+  TooManyFunctions:
+    thresholdInClasses: 18
 
 naming:
   active: true
   FunctionNaming:
     active: true
-    ignoreAnnotated: ['Composable']
+    ignoreAnnotated: [ 'Composable' ]
 
 style:
+  ForbiddenComment:
+    values: [ 'FIXME:', 'STOPSHIP:' ]
```

Comment:
What's wrong with leaving TODO as a forbidden comment?

Reply:
I put it back and added `https://github.com/bumble-tech/appyx/issues/*` as an allowed pattern